lib/fs.js

```diff
@@ -1620,22 +1621,57 @@ function lstat(path, options = { bigint: false }, callback) {
 /**
  * Asynchronously gets the stats of a file.
  * @param {string | Buffer | URL} path
- * @param {{ bigint?: boolean; }} [options]
+ * @param {{ bigint?: boolean, signal?: AbortSignal }} [options]
  * @param {(
  *   err?: Error,
  *   stats?: Stats
  *   ) => any} callback
  * @returns {void}
  */
-function stat(path, options = { bigint: false }, callback) {
+function stat(path, options = { bigint: false, signal: undefined }, callback) {
```

Comment:
Since we are here, probably better do
```suggestion
function stat(path, options = { __proto__: null, bigint: false, signal: undefined }, callback) {
```

And also other objects.

Reply:
thanks

Reply:
I thought it was only here, thank you very much @jazelly